utils/land-patterns/pose-arrangement.stanza

```diff
@@ -0,0 +1,127 @@
+defpackage ocdb/land-patterns/pose-arrangement:
+  import core
+  import collections
+  import math
+  import jitx
+  
+defstruct Affine:
```

Comment:
I would move `defstruct Affine` to jitpcb repository such that we can reuse it if needed

Reply:
I would like for everything that can be expressed at the library level to be done so. We can always move things around later, but this is for operations on `Pose`s to compose with other operations and should be exposed as much as possible.